src/components/About.js

```diff
@@ -1,18 +1,38 @@
-import React from "react";
+import React, { useRef, useState, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 
 import textStyles from "./TextSection/text-section.module.css";
 import imgStyles from "../css-modules/photo-text-section.module.css";
```

Comment:
One question: Why do you load the image with background on CSS? I already told you multiple times about the [disadvantages of doing it](https://nystudio107.com/blog/the-css-background-image-property-as-an-anti-pattern). You can use them, but inline, not in the CSS.

Reply:
Ah I forgot about it needing to be inline and tried using object-fit but couldn't understand why it wasn't working. Thanks for reminding me